### ISSUE TITLE
Custom SDXL Model | No connection adapters were found for [MODEL]

### ISSUE THREAD

**Issue by WaGi-Coding** [brycedrennan/imaginAIry#484]
i just wanted to use a SDXL model fro civitAI i use just fine with A1111's WEBUI. But with "imagine" i get:
`No connection adapters were found for 'D:/SDModels/copaxTimelessxlSDXL1_v8.safetensors'`

Thats my full imagine prompt

```
🤖🧠> imagine --model-architecture "sdxl" --model-weights-path "D:/SDModels/copaxTimelessxlSDXL1_v8.safetensors" --prompt-strength 3.5 --image-prompt "./outputs/generated/aa.png" --image-prompt-strength 0.75 "a close up of a person standing in front of a building, cyberpunk art, inspired by Aleksi Briclot, fantasy art, amazing space creature 4 k, peter mohrbacher'', cybernetic dreadlocks, cyberpunk photo"
Generating 1 images. (1 prompt(s) x 1 repetitions)

🖼  1/1 "a close up of a person standing in front of a building, cyberpunk art, inspired by Aleksi Briclot, fantasy art, amazing space creature 4 k, peter mohrbacher'', cybernetic dreadlocks, cyberpunk photo"
    negative-prompt:DEFAULT-NEGATIVE-PROMPT
    size:1024x1024px-(1:1) seed:156110388 prompt-strength:3.5 steps:50 solver-type:ddim init-image-strength:0.2 arch:sdxl weights:custom
    Loading SDXL weights from D:/SDModels/copaxTimelessxlSDXL1_v8.safetensors
    Loaded SDXL pipeline from D:/SDModels/copaxTimelessxlSDXL1_v8.safetensors (in 0.7ms)
No connection adapters were found for 'D:/SDModels/copaxTimelessxlSDXL1_v8.safetensors'
```

What am I missing and/or doing wrong? ._.

**Comment by davidiwharper** [comment]
I've also run into this error with SD 1.5. As a test, I converted a checkpoint to Diffusers format and specified a directory instead of the original Safetensors file, also to no avail.

**Comment by iwasrobert** [comment]
As a workaround, use Huggingface:

[code]
imagine --model-weights-path https://huggingface.co/XpucT/Deliberate/resolve/main/Deliberate_v6.safetensors --model-architecture sd15 "a flower"
[/code]

This downloads the model, converts it to diffusers, and then you can generate normally.

You can permanently add the model by updating site-packages/imaginairy/config.py:

[code]
    ModelWeightsConfig(
        name="RealVis XL 4.0",
        aliases=["realistic-vision-xl"],
        architecture=MODEL_ARCHITECTURE_LOOKUP["sdxl"],
        defaults={
            "negative_prompt": "(worst quality, low quality, illustration, 3d, 2d, painting, cartoons, sketch), open mouth",
            "composition_strength": 0.6,
        },
        weights_location="https://huggingface.co/SG161222/RealVisXL_V4.0/resolve/main/RealVisXL_V4.0.safetensors",
    ),
[/code]

Run 'imagine' like so to download and install the model for later re-use:

imagine --model realistic-vision-xl "a flower"

**Comment by brycedrennan** [comment]
thank you. For sdxl at least, this is fixed now in 14.3.0
